lib/trace-writer.js

```diff
@@ -17,10 +17,15 @@
 'use strict';
 
 var utils = require('./utils.js');
+var constants = require('./constants.js');
+var pjson = require('../package.json');
 
 /* @const {Array<string>} list of scopes needed to operate with the trace API */
 var SCOPES = ['https://www.googleapis.com/auth/trace.append'];
```

Comment:
It is not exported or used in any other file, so there is no benefit from moving it to constants.